docs/repositories-configure/managing-branches.md

```diff
@@ -1,12 +1,9 @@
 # Managing branches
 
 {% if not extra.self_hosted %}
-Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
+Just like on your Git provider, every repository on Codacy has a default branch. Codacy always analyzes the default branch and loads its data first on dashboards. When you add a repository, the default branch on Codacy matches the default branch on your Git provider (typically `master` or `main`) and is the only one being analyzed.
```

Comment:
* I realized that here we're unclear on what a Codacy default branch is, vs. a Git provider default branch. I tried to clarify.
* Moreover, I found it hard to include the mention that Codacy analyzes multiple branches without sounding like it was just shoved in there. The whole paragraph is about default branches, and the next line introducing the steps should make it fairly clear that you can analyze other branches. I get that we may want to "push" the feature here a bit, but it still feels a bit off...

Reply:
Moreover:

* I included the bit about loading on dashboards, taken from the admonition.
* I removed the mention "consider matching the default on your provider to the default on Codacy." The default already does that, and I'm not sure there's much value. It's an edge case for which, hopefully, a user can notice what's going on independently. Just a guess though.

Reply:
This is a bigger edit and potentially out of scope for this task, but it just naturally came to light as I was editing. I'm open to moving this one to another task.

Reply:
Maybe we're all overthinking this a bit :sweat_smile:, but I find that the new paragraph is more complex than the previous one.

I have a suggestion on how it could be simplified, but I agree that this larger edit deserves its own pull request. This way, we can already merge everything related to the "recommends" and we can fine-tune the "Managing branches" page separately.